README.md

```diff
@@ -313,6 +313,11 @@ configuration is supported at the moment.
 
 Similar to `controller`, the `parent` references the connection profile in the ansible
 role.
+Here is how to specify the VLAN ID
+
+```yaml
+type: vlan
+vlan_id: 123
```

Comment:
The vlan id was already deprecated since https://github.com/linux-system-roles/network/commit/72f7c8ab4dd7d79b2a86bc38ef04382f0705461e,  maybe you can change it into:

The vlan id can be specified by using nested `vlan` setting, the valid vlan id value ranges from 0 to 4094.
```
    vlan:
        id: 6
```

Reply:
Thank you so much for the correction, I’ll correct it and commit again. Thank you!